lib/core/jtrace.c

```diff
@@ -648,6 +1107,50 @@ j_trace_leave(JTrace* trace)
 		G_UNLOCK(j_trace_echo);
 	}
 
+	if (j_trace_flags & J_TRACE_ACCESS)
+	{
+		if (trace_thread->access.inside)
+		{
+			if (strncmp(trace->name, "backend_", 8) == 0)
```

Comment:
possibly split the string instead of using the lengths of expected substrings. see above

Reply:
Splitting would remove the possibility of using longer prefixes, have introduced a `J_TRACE_ACCESS_PREFIX` to remove magic numbers and recurrent string constants